WordPress/src/main/java/org/wordpress/android/networking/GravatarApi.java

```diff
@@ -104,14 +104,20 @@ public void run() {
 
                     @Override
                     public void onFailure(okhttp3.Call call, final IOException e) {
-                        Map<String, Object> properties = new HashMap<>();
-                        properties.put("network_exception_class", e != null ? e.getClass().getCanonicalName() : "null");
-                        properties.put("network_exception_message", e != null ? e.getMessage() : "null");
-                        AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_UPLOAD_EXCEPTION, properties);
-                        CrashlyticsUtils
-                                .logException(e, AppLog.T.API, "Network call failure trying to upload Gravatar!");
-                        AppLog.w(AppLog.T.API, "Network call failure trying to upload Gravatar!" + (e != null
-                                ? e.getMessage() : "null"));
+                        // Don't track exceptions caused by poor internet connectivity
+                        if (!(e instanceof java.net.UnknownHostException)) {
+                            Map<String, Object> properties = new HashMap<>();
+                            String exceptionClass = e != null ? e.getClass().getCanonicalName() : "null";
+                            String exceptionMessage = e != null ? e.getMessage() : "null";
+
+                            properties.put("network_exception_class", exceptionClass);
+                            properties.put("network_exception_message", exceptionMessage);
+                            AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_UPLOAD_EXCEPTION, properties);
+                            CrashlyticsUtils
+                                    .logException(e, AppLog.T.API, "Network call failure trying to upload Gravatar!");
+                            AppLog.w(AppLog.T.API, "Network call failure trying to upload Gravatar!"
```

Comment:
I think it would still make sense to leave an AppLog log with the error, even if we don't bump the Track or send the exception to Crashlytics, so to have better info during Customer Support. WDYT @jkmassel?

Reply:
This makes complete sense – I've updated the PR to reflect this change!